### DISCUSSION TITLE
A simple question about IP configuration

### DISCUSSION THREAD

**Discussion by Sticcolet** [srsran/srsRAN_Project#852] [General Help]
I have a simple question. Is' bind_addr 'actually the port that srsgnb listens on in the configuration file? I ran AMF in the Docker container of the host and set the AMF's addr in the configuration file to the container's IP. However, I found that although the SCTP connection on port 38412 of the AMF IP in the CLI works, when I set the bind_addr in the configuration file to the host address, such as 127.0.0.1, the srsgnb program cannot connect to AMF and can only be used if it is set to 0.0.0.0. Why is this?

**Comment by pgawlowicz** [comment]
If you run the dockerized Open5gs from our repository, it runs with `addr: 10.53.1.2`, therefore you need to set the `bind_addr: 10.53.1.1`.
